components/Buttons/FavSeenContainer/ButtonSeen/index.js

```diff
@@ -3,8 +3,51 @@ import { ButtonSeen, ButtonSeenStyle } from "./ButtonsStyle";
 import CheckSeen from "./check-seen.svg";
 import Image from "next/image";
 import React, { useState, useEffect } from "react";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import styled from "styled-components";
+import { ToastContainer, ProgressBar } from "react-toastify";
 
-const SeenButton = ({ movieId, isSeen, toggleSeen }) => {
+const StyledToastContainer = styled(ToastContainer)`
+  .Toastify__toast {
+    background-image: linear-gradient(
+      to left,
+      #400000,
+      #520602,
+      #631104,
+      #731a03,
+      #842400
+    );
+    color: #ffb833;
+    border: solid #ffb833;
+    border-bottom: solid #d68c29;
+    border-width: 2px;
+    border-radius: 20px;
+    box-shadow: 1px 5px 2px 1px rgba(70, 0, 0, 0.15);
+    width: -50%;
+  }
+
+  .Toastify__progress-bar--controlled {
+    background-color: gold;
+  }
+
+  .Toastify__emoji {
+    display: inline-block;
+    font-size: 16px;
+    margin-right: 4px;
+  }
+`;
```

Comment:
This is code I just saw the second time. Would it be a good idea creating a component out of it and use it here again?